test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs

```diff
@@ -32,7 +33,7 @@ public void RunTest_DoesNotRunOnWin7OrWin2008R2()
         {
             Assert.False(
                 RuntimeInformation.IsOSPlatform(OSPlatform.Windows) &&
-                Microsoft.Extensions.Internal.RuntimeEnvironment.OperatingSystemVersion.StartsWith("6.1"),
+                Environment.OSVersion.Version.ToString().StartsWith("6.1"),
```

Comment:
@BrennanConroy can we use netstandard2.0 then? As this isn't a library we ship, I think we're not required to restrict this to netcoreapp2.0 only.